examples/sklearn-logreg-mnist/README.md

```diff
@@ -4,83 +4,64 @@ dataset: [MNIST]
 framework: [scikit-learn]
 ---
 
-# Flower Logistic Regression Example using scikit-learn
+# Flower Logistic Regression Example using scikit-learn and Flower (Quickstart Example)
 
 This example of Flower uses `scikit-learn`'s `LogisticRegression` model to train a federated learning system. It will help you understand how to adapt Flower for use with `scikit-learn`.
```

Comment:
```suggestion
This example of Flower uses `scikit-learn`'s [LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) model to train a federated learning system. It will help you understand how to adapt Flower for use with `scikit-learn`.
```

Reply:
likely some extra details can be added in the intro paragraph. I thought of at least linking to the linear regression docs page.